.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,32 @@
+name: Dependabot auto-approve-merge
+
+# yamllint disable-line rule:truthy
```

Comment:
can you quote value instead?

Reply:
hmm....not sure if you can quote the ``on`` but I will try

Reply:
cool.  Added quotes worked